sdks/python/apache_beam/typehints/decorators_test_py3.py

```diff
@@ -33,18 +34,20 @@
 
 decorators._enable_from_callable = True
 T = TypeVariable('T')
+# Name is 'T' so it converts to a beam type with the same name.
+T_typing = typing.TypeVar('T')
 
 
 class IOTypeHintsTest(unittest.TestCase):
 
   def test_from_callable(self):
     def fn(a: int, b: str = None, *args: Tuple[T], foo: List[int],
-           **kwargs: Dict[str, str]) -> Tuple:
+           **kwargs: Dict[str, str]) -> Tuple[Any, ...]:
```

Comment:
These should be synonyms, right?

Reply:
Yes if these were typing module types, but in this case this is `typehints.Tuple[typehints.Any, ...]`, which requires subscripting to turn it into a constraint. I will remove such usage in the future and make the `typehints.` prefix explicit.

FYI, the conversion from typing.Tuple to a typehints.Tuple constraint is taken care of here: https://github.com/apache/beam/pull/10042/files#diff-9b2eac354738047adddd44814d4b429cR245